src/app/story/storyrow/storyrow.component.html

```diff
@@ -1,4 +1,6 @@
-<div class="node-uid-column story-row-column">{{ node.display_name | nodeDisplayName | truncate: 30 }}</div>
+<div class="node-uid-column story-row-column">{{ node.display_name | nodeDisplayName | truncate: 30 }}<br>
```

Comment:
Instead of using break statements, the preferred approach would be to use two inner divs. Since divs have `display: block` by default, you should still get the line break, but you might have to modify the margins.